src/backend/vulkan/src/conv.rs

```diff
@@ -623,3 +623,72 @@ pub fn map_vk_memory_heap_flags(flags: vk::MemoryHeapFlags) -> hal::memory::Heap
 
     hal_flags
 }
+
+pub fn map_vk_surface_transform_flags(
+    vk_flags: vk::SurfaceTransformFlagsKHR,
```

Comment:
Sure, done. Since the conversion can be done by a simple `from_raw`/`from_bits`, i have also removed the related conv functions because they are no more needed.